cmd/core-metadata/main.go

```diff
@@ -48,20 +51,28 @@ func main() {
 	flag.Usage = usage.HelpCallback
 	flag.Parse()
 
-	dic := di.NewContainer(di.ServiceConstructorMap{})
+	configuration := &config.ConfigurationStruct{}
+	dic := di.NewContainer(di.ServiceConstructorMap{
+		container.ConfigurationName: func(get di.Get) interface{} {
+			return configuration
+		},
+		bootstrapContainer.ConfigurationInterfaceName: func(get di.Get) interface{} {
+			return get(container.ConfigurationName)
+		},
```

Comment:
Please remove this.  See https://github.com/edgexfoundry/edgex-go/issues/2206 merged earlier today.

Reply:
Done. Rebased and ready for review 👍